### PR TITLE
Safely add attribute values to prevent XSS

### PR DESCRIPTION
**Pull request by kofrezo**

### CODE REVIEW

File: serveradmin/servershell/static/js/servershell/result.js [hunk 1]
```diff
@@ -84,24 +84,40 @@ get_row_html = function(object, number) {
                     let to_delete = change.remove.join(', ');
                     let value = object[attribute_id].filter(v => !to_delete.includes(v)).join(', ');
 
-                    cell.html(`${value} <del>${to_delete}</del> <u>${to_add}</u>`)
+                    cell.text(value);
+
+                    let del = $('<del></del>');
```
Comment (TheFRedFox) [inline serveradmin/servershell/static/js/servershell/result.js:89]: Btw, for creating just an empty html elements you don't need the closing tag:
``` js
$('<del>')
```
is sufficient already.

File: serveradmin/servershell/static/js/servershell/result.js [hunk 2]
```diff
@@ -84,24 +84,40 @@ get_row_html = function(object, number) {
...
+                    cell.append(del);
+
+                    let add = $('<u></u>');
+                    add.text(to_add);
+                    cell.append(add);
                 }
                 else {
                     let to_delete = change.old === null ? '' : change.old;
                     let new_value = change.new === undefined ? '': change.new;
-                    cell.html(`<del>${to_delete}</del>&nbsp;<u>${new_value}</u>`);
```
Comment (TheFRedFox) [inline serveradmin/servershell/static/js/servershell/result.js:89]: I'm not sure, but maybe you will use that `&nbsp;` whitespace. Is that ok?

**Review by TheFRedFox — APPROVED** [review]
Just 2 minor discussions. Nothing for rejection. Already approve.

File: serveradmin/servershell/static/js/servershell/result.js (hunk 1 above)
Comment (kofrezo) [reply serveradmin/servershell/static/js/servershell/result.js:89]: Ah neat. I will adjust the code.

File: serveradmin/servershell/static/js/servershell/result.js (hunk 2 above)
Comment (kofrezo) [reply serveradmin/servershell/static/js/servershell/result.js:89]: When including the `&nbsp` the jQuery methods fail so I decided to remove it. We should do it with CSS anyways if it bothers people but I think it looks ok.

**Pull request merged**